components/Picker.jsx

```diff
@@ -0,0 +1,221 @@
+/**
+ * Picker.jsx
+ * Created by Kevin Li 8/16/17
+ */
+
+import React, { createRef, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+require('../styles/components/_picker.scss');
+
+const fontAwesomeIconId = "usa-dt-picker__button-icon--svg";
+
+const propTypes = {
+    sortFn: PropTypes.func,
+    icon: PropTypes.string,
+    altText: PropTypes.string,
+    iconColor: PropTypes.string,
```

Comment:
Some of these props could be achieved with css overrides instead (colors, border) - I would lean towards keeping the component simpler and bare-bones functionality.

Reply:
Agreed -- removed all of em!